tests/Symfony/Upgrade/Fixtures/Fixer/form-type-names/case1-output-php54.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
Rename this file to `case2-output.php` and create copy of `case1-input.php` > `case2-input.php` to avoid changes in `FormTypeNamesFixerTest`.